### PR TITLE
rename license to conform to spdx

### PR DESCRIPTION
**Pull request by khusmann**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

@conda-forge-admin please rerender

### CODE REVIEW

**Comment by khusmann** [comment]
@conda-forge-admin, please rerender

**Comment by khusmann** [comment]
@conda-forge-admin, please re-render

**Comment by khusmann** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,11 +1,3 @@
-appveyor:
```
Comment (dbast) [inline conda-forge.yml]: Why is this all removed?... win+osx do not run here because the recipe is noarch.

Comment (khusmann) [reply conda-forge.yml]: ah, that makes sense. this was a shot in the dark to try to understand why another package was failing that I'm working on  (https://github.com/conda-forge/staged-recipes/pull/12143) When I updated the spdx here I noticed the win+osx CIs weren't being run and interpreted it to mean that the win+osx packages weren't being updated for some reason.

noarch makes sense! I'll revert to the first commit then.

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Pull request merged**